client/src/components/Header/Header.test.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import Header from './Header'
+
+describe('Header', () => {
+  it('renders without crashing', () => {
+    const { container } = render(<Header />)
+    expect(container.firstElementChild).toBeInTheDocument()
+  })
+
+  it('renders a Header', () => {
+    const { container } = render(<Header />)
+    expect(container.firstElementChild!.tagName).toBe('HEADER')
+  })
+
+  it('renders the title prop', () => {
+    const { container } = render(<Header title='Hello world' />)
+    const title = container.querySelector('h1')
```

Comment:
According to the guiding principles of `@testing-library`
> your test should resemble how users interact with your code (component, page, etc.) as much as possible

In this case, the user will find the title not by its HTML element type but by the text it contains. And so, should our test using `getByText` instead of a `querySelector`

Reply:
✅